pyproject.toml

```diff
@@ -1,3 +1,22 @@
+[tool.black]
+line-length = 99
+target-version = ['py310']
+
+[tool.django-stubs]
+django_settings_module = "django_webapps.settings"
+
+[tool.isort]
+profile = "black"
+line_length = 99
+skip_glob = ["api/migrations/*"]
+
+[tool.mypy]
+exclude = "api/migrations"
+plugins = ["mypy_django_plugin.main", "mypy_drf_plugin.main"]
+python_version = "3.10"
+strict = true
+strict_optional = true
+
```

Comment:
I recommend to add
`ignore_missing_imports = true`
https://mypy.readthedocs.io/en/stable/config_file.html#confval-ignore_missing_imports

to avoid using type:ignore in cases such

`from phonenumber_field import modelfields  # type: ignore`

Reply:
OK

Reply:
My reasoning was that right now this `type: ignore` thing is only needed in one place while this setting will ignore all missing imports which may not be a good thing. Like, what if stubs are available and we don't go looking for them because mypy gave no warning?